app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -201,6 +202,12 @@ abstract class AppDatabase : RoomDatabase() {
         purgeDeleted()
         purgeDeletedPhoneNumbers()
         purgeDeletedBusinessIds()
+        purgeDeletedPatientAfterRetentionTime(Instant.now())
```

Comment:
We are creating an `Instant` without a `Clock` here. Please use the `Clock` to create an `Instant`, always.

Reply:
@msasikanth We should consider adding a lint check for creating date and time instances without the `Clock`.

Reply:
Yeah, makes sense 👍🏾

Reply:
Also @Honey14 we can probably move this param to the `purge` method so that we can override the current time in tests as well.